python/src/opendp/_extrinsics/_make_np_eigenvector/__init__.py

```diff
@@ -75,7 +75,7 @@ def function(C):
             #    2. compute clamp_norm(compute L @ std_gaussian(d), 1) with arbitrary precision
             #       sample with sufficient precision where all components round to same float
             
-            z = np_csprng.multivariate_normal(mean=np.zeros(d), cov=Omega_inv)
+            z = np_csprng.multivariate_normal(mean=np.zeros(d), cov=Omega_inv)  # type: ignore[union-attr] 
```

Comment:
Is this necessary? I don't see changes in this PR that are obviously related.

Reply:
Strange, my MyPy fails if I don't have it.